src/style.css

```diff
@@ -1 +1,79 @@
-/* Styles go here */
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  --color: #00acdc;
```

Comment:
it is better to give more descriptive names for your variables. For colors we usually use names like `color-primary` or `primary-blue` or sth. Just keep that in mind next time